it_IT/contributing.md

```diff
@@ -1,7 +1,7 @@
 ___
 **Nota del traduttore**
 
-Questa è la traduzione del file [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Qui c'è il [link](https://github.com/sindresorhus/ava/compare/f2c070987ecee3caf7613190acf2c8a90700e058...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9) dove si confrontano le differenze tra commit di riferimento di questa traduzione e l'ultimo commit di AVA sul branch master (Se si clicca sul link, e non si vede il file `contributing.md` nella lista dei file modificati, questa traduzione è aggiornata).
+Questa è la traduzione del file [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Qui c'è il [link](https://github.com/sindresorhus/ava/compare/3201b1b4ff80ff75f0e1c288ca7da22f92c9b814...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9) dove si confrontano le differenze tra commit di riferimento di questa traduzione e l'ultimo commit di AVA sul branch master (Se si clicca sul link, e non si vede il file `contributing.md` nella lista dei file modificati, questa traduzione è aggiornata).
```

Comment:
I see. I used instead a bulk approach:

• pick latest commit
• translate all the files up to that
• change the compare link to that commit

I'll update the link tonight.

Marco

> On 12 apr 2016, at 08:41, forresst <notifications@github.com> wrote:
> 
> In it_IT/contributing.md:
> 
> > @@ -1,7 +1,7 @@
> >  ___
> >  **Nota del traduttore**
> >  
> > -Questa è la traduzione del file [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Qui c'è il [link](https://github.com/sindresorhus/ava/compare/f2c070987ecee3caf7613190acf2c8a90700e058...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9) dove si confrontano le differenze tra commit di riferimento di questa traduzione e l'ultimo commit di AVA sul branch master (Se si clicca sul link, e non si vede il file `contributing.md` nella lista dei file modificati, questa traduzione è aggiornata).
> > +Questa è la traduzione del file [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Qui c'è il [link](https://github.com/sindresorhus/ava/compare/3201b1b4ff80ff75f0e1c288ca7da22f92c9b814...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9) dove si confrontano le differenze tra commit di riferimento di questa traduzione e l'ultimo commit di AVA sul branch master (Se si clicca sul link, e non si vede il file `contributing.md` nella lista dei file modificati, questa traduzione è aggiornata).
> Yes but the last commit sindresorhus/ava@3201b1b not contains the file contributing.md
> 
> The last commit the affected file that you translated, is very useful for future translations. It helps to know where you should resume translations if this file is changed.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>